pages/about/research/recruitment/general-fed.md

```diff
@@ -0,0 +1,53 @@
+---
+permalink: /about/research/recruitment/general-fed/
+layout: styleguide
+title: How the federal USWDS community can participate in our research
+category: About
+lead: Help shape the future of U.S. government websites.
+redirect_from:
+  - /research/recruitment/federal/
+changelog:
+  key: about-research-recruitment-federal
+subnav: true
+---
+
+How the federal USWDS community can participate in our research
+Getting feedback from designers, developers, UX practitioners, content strategists, and others in the USWDS community is critical for us to deliver a design system that works for you.
+
+We regularly carry out research to better understand what you need from the design system and to get early feedback on ideas we have.
+
+## Sign up to be added to our list of participants
+
+{:.border-top-2px .border-base-lighter .padding-top-1}
+
+We’re looking for federal government employees who either currently use USWDS or who are curious about USWDS. We unfortunately can’t include contractors as part of this research effort.
+
+**To sign up:**  Fill out a brief research signup form or email us at <{{ site.uswds_email }}> to tell us you’re interested. We’ll reach out when we have a research opportunity for you.
+
+Your participation is completely voluntary and you can opt out at any time by emailing us at <{{ site.uswds_email }}>.
+
+## What to expect if we invite you to participate
+
+{:.border-top-2px .border-base-lighter .padding-top-1}
+
+We’ll share the research purpose and details with you when we reach out to see if you’re interested in participating. Typically, participating could mean:
+
+- Offering asynchronous feedback on a concept
+- Having a 30-60 minute conversation with us
+- Testing a prototype and thinking out loud (which we would ask to record)
```

Comment:
Wondering how this phrasing might affect non-verbal participants. Just a thought.